src/solc_0.7/common/Interfaces/GameToken.sol

```diff
@@ -0,0 +1,99 @@
+//SPDX-License-Identifier: MIT
```

Comment:
the filename should be renamed to `GameTokenInterface.sol`

Reply:
updated.